### ISSUE TITLE
CICI build error

### ISSUE THREAD

**Issue by kalondar** [xaaskit/cdk-aws-lambda-dotnet#5]
Everything deploys from my local machine but failed when running on CircleCi

```
unable to prepare context: unable to evaluate symlinks in Dockerfile path: CreateFile C:\Users\circleci.PACKER-6400C91A\calculation-module-cloud\node_modules\@xaaskit-cdk\aws-lambda-dotnet\lib\Dockerfile: The system cannot find the file specified.
C:\Users\circleci.PACKER-6400C91A\calculation-module-cloud\node_modules\aws-cdk-lib\core\lib\private\asset-staging.js:2
`).map((line,idx)=>`${idx===0?firstLine:padding}${line}`)};const reason=proc.signal!=null?`signal ${proc.signal}`:`status ${proc.status}`,command=[prog,...args.map(arg=>/[^a-z0-9_-]/i.test(arg)?JSON.stringify(arg):arg)].join(" ");throw new Error([`${prog} exited with ${reason}`,...prependLines("--> STDOUT:  ",proc.stdout)??[],...prependLines("--> STDERR:  ",proc.stderr)??[],`--> Command: ${command}`].join(`
                                                                                                                                                                                                                                            ^
Error: docker exited with status 1
--> Command: docker build -t cdk-f3acee45d8ea3cdb9df58090ba4c8a58e621107d3f63bc86a1216638c95ea036 --platform "linux/amd64" --build-arg "IMAGE=public.ecr.aws/sam/build-dotnet6" "C:\\Users\\circleci.PACKER-6400C91A\\calculation-module-cloud\\node_modules\\@xaaskit-cdk\\aws-lambda-dotnet\\lib\\"
    at dockerExec (C:\Users\circleci.PACKER-6400C91A\calculation-module-cloud\node_modules\aws-cdk-lib\core\lib\private\asset-staging.js:2:237)
    at Function.fromBuild (C:\Users\circleci.PACKER-6400C91A\calculation-module-cloud\node_modules\aws-cdk-lib\core\lib\bundling.js:1:4085)
    at new Bundling (C:\Users\circleci.PACKER-6400C91A\calculation-module-cloud\node_modules\@xaaskit-cdk\aws-lambda-dotnet\src\bundling.ts:70:25)
    at Function.bundle (C:\Users\circleci.PACKER-6400C91A\calculation-module-cloud\node_modules\@xaaskit-cdk\aws-lambda-dotnet\src\bundling.ts:33:22)
```

Notice that `@xaaskit-cdk\\aws-lambda-dotnet\\lib\\` was passed as a build directory instead of one from `projectDir/solutionDir` attribute.

Besides... `src/Dockerfile` is not bundled into Node package

**Comment by jeastham1993** [comment]
Just to add to this, I'm seeing the same issue using GitHub Actions.

**Comment by vlesierse** [comment]
An alternative would be to install `esbuild` into you action. I just released 0.0.10 which should now include the `Dockerfile`.

**Comment by kalondar** [comment]
V0.0.11 still doesn't contain the Dockerfile:

<img width="259" alt="image" src="https://github.com/xaaskit/cdk-aws-lambda-dotnet/assets/523176/16ea7469-dc39-4621-b9d4-5399791f4cd5">

Still the same error.

Hmm - there is a Dockerfile inside `src` directory, but the build try to find it in `lib`.

**Comment by vlesierse** [comment]
It turned out that I can remove the Dockerfile all together as the default runtime image contains the .NET Lambda Powertools. Hope this helps.

**Comment by vlesierse** [comment]
Fixed in release [v0.13.0](https://github.com/xaaskit/cdk-aws-lambda-dotnet/releases/tag/v0.0.13)
